static/scss/dashboard.scss

```diff
@@ -143,7 +143,6 @@
       font-size: 11pt;
       align-items: center;
       padding: 14px 20px;
-      border-bottom: 1px solid $lightgrey;
 
       &:last-child {
```

Comment:
@noisecapella actually, since we're no longer putting a border on `.course-row`, this `&:last-child` block can probably go too.